src/libutil/experimental-features.cc

```diff
@@ -37,20 +139,40 @@ const std::optional<ExperimentalFeature> parseExperimentalFeature(const std::str
         return std::nullopt;
 }
 
-std::string_view showExperimentalFeature(const ExperimentalFeature feature)
+std::string_view showExperimentalFeature(const ExperimentalFeature tag)
 {
-    const auto ret = get(stringifiedXpFeatures, feature);
-    assert(ret);
-    return *ret;
+    assert((size_t)tag < xpFeatureDetails.size());
+    return xpFeatureDetails[(size_t)tag].name;
+}
+
+std::string getExperimentalFeaturesList() {
+    std::string experimentalFeaturesList = R"(
+        Experimental Nix features to enable.
+        Current experimental features are the following:
+
+)";
+
+    for (auto & xpFeature : xpFeatureDetails) {
+        experimentalFeaturesList += std::string {}
+            /* length of this first string must be 12, matching the indent of
+               the descriptions in the xpFeatureDetails literal. FIXME compute
+               markdown in a less hacky way. */
```

Comment:
It is probably better to just split out JSON and assemble the markdown bullets out-of-band, but I am OK doing that in a follow-up PR

Reply:
Actually, I am less OK. This can get in after the experimental features documentation #5930 so that gives me some time to improve this.

Reply:
@Ericson2314 maybe open an issue to keep track of it then? I'll resolve this because it doesn't relate to getting this PR merged.